payments-core/src/main/java/com/stripe/android/payments/bankaccount/ui/CollectBankAccountActivity.kt

```diff
@@ -0,0 +1,35 @@
+package com.stripe.android.payments.bankaccount.ui
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.lifecycleScope
+import com.stripe.android.payments.bankaccount.navigation.CollectBankAccountContract
+
+/**
+ * No-UI activity that will handle collect bank account logic.
+ */
+internal class CollectBankAccountActivity : AppCompatActivity() {
+
+    private val starterArgs: CollectBankAccountContract.Args? by lazy {
+        CollectBankAccountContract.Args.fromIntent(intent)
+    }
+
+    internal var viewModelFactory: ViewModelProvider.Factory =
+        CollectBankAccountViewModel.Factory(
+            { application },
+            { requireNotNull(starterArgs) },
+            this,
+            intent?.extras
+        )
+
+    private val viewModel: CollectBankAccountViewModel by viewModels { viewModelFactory }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        lifecycleScope.launchWhenStarted {
+            viewModel.viewEffect.collect {}
```

Comment:
Can we put a todo here? I think it will help us understand what we need to do after.